test/integration/webhook/v1alpha2/clusterqueue_test.go

```diff
@@ -64,7 +66,7 @@ var _ = ginkgo.Describe("ClusterQueue Webhook", func() {
 		ginkgo.It("Should have qualified resource names when creating", func() {
 			ginkgo.By("Creating a new clusterQueue")
 			cq := testing.MakeClusterQueue("cluster-queue").Resource(
-				testing.MakeResource("@cpu").Obj(),
+				testing.MakeResource("@cpu").Flavor(testing.MakeFlavor("x86", "5").Obj()).Obj(),
```

Comment:
we should do this if we don't update `MakeResource` function.

Reply:
make sure there is an integration test that fails on empty flavors